main.js

```diff
@@ -160,4 +160,141 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+//OPDRACHT 1
+
+function getTVTypeNames() {
+  let types = [];
+  inventory.forEach((tv) => {
+    types.push(tv.type);
+  });
+  return types;
```

Comment:
Een makkelijkere manier is om de `.map` functie te gebruiken:
`return inventory.map(tv => tv.type);`